src/pages/generate/hooks/useGenerate.ts

```
@@ -187,32 +187,50 @@ export const useGenerateImageApi = () => {
 
   const query = useQuery({
-    queryKey: ['generateImageStatus', houseId],
-    queryFn: () => getCheckGenerateImageStatus(houseId),
-    enabled: shouldStart,
-    refetchInterval: 7000, // 7초
-    refetchIntervalInBackground: true,
-    retry: (failureCount) => {
-      // 최대 10번 재시도
-      if (failureCount >= 9) {
-        console.error('최대 재시도 횟수 초과');
+    queryKey: ['fallbackImage', houseId],
```

ESLint

[error] Replace `'fallbackImage'` with `"fallbackImage"`

(prettier/prettier)